### ISSUE TITLE
Help buttons should go to specific articles, not general docs page

### ISSUE THREAD

**Issue by herkalurk** [Ombi-app/Ombi#4223]
**Describe the bug**
I'm in the settings window for Discord notifications. At the top right of that windows is a button that says "Notification Variables' but when I click on it, I get directed to the general docs. This button should go direct to the variables page without searching. I'm guessing this isn't the only place in the settings where this is occurring.

**To Reproduce**
1. Go to discord settings
2. Click on 'Notification Variables' button
3. Resulting web page is generic docs

**Expected behavior**
Help buttons go to articles specific to topic

**Ombi Version (please complete the following information):**
 - Version 4.0.1342 (Docker container)
- Media Server Plex
- Database Type: SQLite

**Comment by Berserkir-Wolf** [comment]
All docs pages for settings have been standardised with Ombi pages in preparation for this (your_ombi_address/settings/mobile would have a docs page of docs.ombi.app/settings/mobile, for example).

Next stage is to adjust the "docs_button" function in Ombi itself to go to docs_site/path based on current path in the UI.
Once that function is in place, any new things added (say new notification methods), then making the page for that method would instantly have a relevant link in ombi.

I've also already created placeholder pages for anything I've not written docs for as well, so all the links will actually work.

@tidusjar - that make sense as a solution for this one (and any future docs pages)?

**Comment by tidusjar** [comment]
Yep that makes sense!

**Comment by Berserkir-Wolf** [comment]
Now to add a docs button to every settings page 😂

One note - [here](https://github.com/Ombi-app/Ombi/commit/197fc5af61c5afb663403dad8396f135593621b2#diff-e85875f142bc8e563a899cce9ad6e9115d5af7b509245e822ceddb251272b8bcR3) 
The link [https://docs.ombi.app/settings/massemail](https://docs.ombi.app/settings/massemail) works, and redirects to [https://docs.ombi.app/settings/notifications/massemail](https://docs.ombi.app/settings/notifications/massemail).  
This matches the URL that Ombi actually presents - ombiurl/settings/massemail.  
Ideally we just need to have the docs buttons point to 'wikilink/activepagepath' based on 'ombiurl/activepagepath'.  
No manual entry needed per page.  

We would of course need to account for baseurl, but in theory the button should just replace applicationurl with wikiurl and it would work (if application url is set).

**Comment by tidusjar** [comment]
Just seen this @Berserkir-Wolf 

So we want to pass it to (for example) `wikilink/settings/massemail` instead?

**Comment by Berserkir-Wolf** [comment]
Yeah - mkdocs handles the redirect to settings/notifications/massemail.
This way anything added going forward just... has a link. If Ombi generates the link based on what page it's showing, then the link is always correct - if it ever mismatches then we amend the docs to catch it.

settings/sonarr redirects to settings/all-the-arrs, for example. So ombi/settings/sonarr could generate wikilink/settings/sonarr, and the user would get shifted to docs/settings/all-the-arrs automatically. 
If a page gets broken out into its own page, we remove the redirect, and the new page goes online.
